library/fsharp

```diff
@@ -2,14 +2,14 @@ Maintainers: Dave Curylo <dave@curylo.org> (@ninjarobot),
              Steve Desmond <steve@stevedesmond.ca> (@stevedesmond-ca)
 GitRepo: https://github.com/fsprojects/docker-fsharp.git
 
-Tags: latest, 4, 4.1, 4.1.18
+Tags: latest, 4, 4.1, 4.1.25
+GitCommit: 96cd7752113e7b4e32fbd6437600816f4b361994
+Directory: 4.1.25/mono
+
+Tags: 4.1.18
 GitCommit: ad4c4d67a1975b2b0ff0acc49dc46b3271836831
 Directory: 4.1.18/mono
 
-Tags: 4.1.18-wheezy-slim
-GitCommit: 33b5be9aea66ea1d258daf23055fa6d9bcb164ff
-Directory: 4.1.18/mono-wheezy
```

Comment:
Is `wheezy-slim` being removed here on purpose?  Is `4.1.18` being kept on purpose? (with 4.1.25 out, 4.1.18 won't be receiving any updates, will it?)